ui/app/components/secret-edit.js

```diff
@@ -112,8 +112,8 @@ export default Component.extend(FocusOnInsertMixin, WithNavToNearestAncestor, {
     'model.id',
     'mode'
   ),
-  canDelete: alias('model.canDelete'),
```

Comment:
changed the wording here because "update" matches the actual capabilities word you use to make this work and in this context it was helpful to keep this syntax the same while trying to understand how to properly set the conditionals.